### ISSUE TITLE
util-linux 2.29.2 needs zlib dependency

### ISSUE THREAD

**Issue by vanzod** [easybuilders/easybuild-easyconfigs#4899]
It seems that `util-linux-2.29.2-intel-2017a.eb` needs `zlib` as dependency. Here is the error in the log:

```
In file included from disk-utils/fsck.cramfs.c(53):
/usr/include/zlib.h(34): catastrophic error: cannot open source file "zconf.h"
  #include "zconf.h"
                    ^
compilation aborted for disk-utils/fsck.cramfs.c (code 4)
Makefile:7513: recipe for target 'disk-utils/fsck.cramfs.o' failed
make[2]: *** [disk-utils/fsck.cramfs.o] Error 4
```

**Comment by vanzod** [comment]
Fixed with #4900

**Comment by boegel** [comment]
fixed in #4900 and #4998
